contrib/scrape-ec2-prices.py

```diff
@@ -101,12 +110,15 @@
 REGION_NAME_MAP = {
     'us-east': 'ec2_us_east',
     'us-east-1': 'ec2_us_east',
+    'us-east-2': 'ec2_us_east',
```

Comment:
I believe the value should match up to the `api_name` attribute of the region. Therefore, should this be `ec2_us_east_ohio` instead?